From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Mon, 17 Jan 2022 13:52:44 -0500
Subject: build: Detect more flexible musl variants of GNU

* configure.ac: expand *-*-linux-musl to *-*-linux-musl*.

--

GnuPG-bug-id: 5762
Reported-by: Helmut Grohne <helmut@subdivi.de>
Signed-off-by: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 1714dfa..a825310 100644
--- a/configure.ac
+++ b/configure.ac
@@ -605,7 +605,7 @@ if test x"$gl_use_threads" = xno; then
   AC_MSG_NOTICE([generated src/lock-obj-pub.native.h for $host])
 elif test x$cross_compiling = xyes; then
   case $host in
-    *-*-linux-gnu* | *-*-linux-musl)
+    *-*-linux-gnu* | *-*-linux-musl*)
     AC_CHECK_TOOL(OBJDUMP, [objdump])
     if test -n "$OBJDUMP"; then
       lock_obj_h_generated=yes
